web-app/helpers/certificate-designs.ts

```diff
@@ -53,24 +55,33 @@ function wrapText(canvas: Canvas, text: string, x: number, y: number, maxWidth:
       line = testLine.trim();
     }
   }
+  context.fillStyle = fillStyle;
+  context.font = font;
   context.fillText(line, x, y2);
 }
 
 // eslint-disable-next-line max-lines-per-function
-export async function populateDeveloperCert(canvas: Canvas, details: any) {
-  console.log('populateDeveloperCert', { details });
+export async function populateCerts(canvas: Canvas, details: any) {
+  console.log('populateCerts', { details });
   const { tokenId, date, programName, accountName, expiration, programDescription, instructor, programCode } = details;
 
   const gray = '#757575';
   const black = '#000000';
   const blue = '#5F8AFA';
-  // font weight doesn't really work at all!
-  const accountFont = `60px '${fontFamily2}' medium`;
-  const dateFont = `30px '${fontFamily2}' medium`;
-  const descriptionFont = `33px '${fontFamily}' regular`;
-  const tokenIdFont = `30px '${fontFamily2}' medium`;
-  const programFont = `40px '${fontFamily2}' medium`;
-  const titleFont = `64px '${fontFamily}' extraBold`;
+
+  const WIDTH_OF_CERTIFICATE = 950;
```

Comment:
This whole section of math is too confusing. E.g. `WIDTH_OF_CERTIFICATE = 950` but `width = 1080; // width of the image`. Then also why so much division and subtraction etc.  I'll add another comment elsewhere about a possible better approach.